target/linux/ramips/dts/mt7628an_wiznet_wizfi630s.dts

```diff
@@ -23,12 +23,13 @@
 		serial1 = &uartlite;
 	};
 
+
```

Comment:
Please don't add all these empty lines.

Reply:
ups - sorry.

> > WizFi630S had some pins changed in the release version of the board. The run led, wps led and wps button where affected. This patch is correcting this.
> 
> Sure? Why has nobody complained so far? Maybe this is just another revision?

Sure - this pinout is the only one used in mass production.